server/webpack.config.js

```diff
@@ -4,6 +4,9 @@ module.exports = {
   mode: 'production',
   devtool: 'source-map',
   stats: 'errors-only',
+  cache: {
+    type: 'filesystem'
```

Comment:
question (non-blocking):  Why use the file system for caching rather than in memory?

Reply:
From what I've read using filesystem is supposed to be more performant and persist if the container restarts.